packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,289 @@ components:
+                      type: number
+                      description: Bitrate of the profile in bits per second
+                      example: 1000000
+                    playbackId:
+                      type: string
+                      description: Playback ID of the asset
+                      example: ABC123
+        status:
+          type: string
+          description: Status of the asset
+          example: ready
+        originTaskId:
+          type: string
+          description: Task ID of the task that created the asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        sourceAssetId:
+          type: string
+          description:
+            ID of the source asset (root) - If missing, this is a root asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+
+    task:
+      type: object
+      table: task
+      additionalProperties: false
+      properties:
+        id:
+          type: string
+          readOnly: true
+          description: Task ID
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        name:
+          type: string
+          description: Name of the task
+          example: Import video file
+        userId:
+          type: string
+          readOnly: true
+          description: User ID of the task owner
+          example: 66E2161C-7670-4D05-B71D-DA2D6979556F
+        type:
+          type: string
+          description: Type of the task
+          enum:
+            - Import
+            - Export
+          example: Import
+        createdAt:
+          type: number
+          readOnly: true
+          description: Timestamp (in milliseconds) at which task was created
+          example: 1587667174725
+        updatedAt:
+          type: number
+          description: Timestamp (in milliseconds) at which task was updated
+          example: 1587667174725
+        deleted:
+          type: boolean
+          description: Set to true when the task is deleted
+        parentAssetId:
+          type: string
+          description: ID of the source asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        params:
+          type: object
+          additionalProperties: false
+          description: Parameters of the task
+          properties:
+            import:
+              type: object
+              additionalProperties: false
+              description: Parameters for the import task
+              properties:
+                url:
+                  type: string
+                  description: URL of the asset to import
+                  example: https://cdn.livepeer.com/ABC123/filename.mp4
+        output:
+          type: object
+          additionalProperties: false
+          description: Output of the task
+          properties:
+            import:
+              type: object
+              additionalProperties: false
+              description: Output of the import task
+              properties:
+                status:
```

Comment:
The `status` field should be shared by all task types instead. I think it's more of a metadata than an output instead as well, so I think it can be moved to the root. 

Following the Kubernetes API, the status could also be an object that has a `phase` of `Pending/Running/Failed/Completed` instead, and we can move the `updatedAt` to inside the status as well WDYT?

```
status:
  phase: Pending
  updatedAt: 12437328123
  // later we can add more fields if appropriate
```

Reply:
Yeah, I agree, we could move it to the root, I was thinking that different kinds of task in the future could have new types of status phases, but I guess we can add them in the phase enum and support all cases